test/integration/serverless/src/log.js

```diff
@@ -9,7 +9,8 @@ async function myHandler(event, context) {
   await sleep();
   console.error("XXX LOG 2 XXX");
   await sleep();
-  
+  console.error("XXX LOG M12345M XXX");
```

Comment:
I would suggest replacing M12345M with something like `IF THIS STRING APPEARS IN A SNAPSHOT, MASKING IS NOT WORKING` so that it's easier to see in a diff what the expected behavior is.